python/lsst/sconsUtils/tests.py

```diff
@@ -191,3 +195,68 @@ def run(self, fileGlob):
             self._env.Clean(target, self._tmpDir)
 
         return targets
+
+    def runPythonTests(self, pyList):
+        """Add a single target for testing all python files.
+        pyList is a list of nodes corresponding to python test files.
+        The IgnoreList is respected when scanning for entries.
+        If pyList is None, we will use automated test discovery within
+        pytest. Returns a list containing a single target."""
```

Comment:
you mean, for formatting the doc paragraph? You want a 72 character wide `M-x fill-paragraph`?